src/lib/ext-activitylog.js

```diff
@@ -251,6 +264,23 @@ class Controller {
       if (existingLogs.length) {
         this.handleNewLogs(existingLogs);
       }
+
+      if (filterTabId) {
+        // currently menu container is hidden when activity log page is
+        // filtered by tab id
+        this.view.menuContainer.hidden = true;
```

Comment:
As the menu container contains save and clear logs button, I am hiding those. 
Since in the devtools panel, the user will see the filtered logs by tabId but when he presses the "save logs" button, it will save all the logs. So, having that button displayed may mislead the user. Same scenario with "clear logs" button too (As it also clears all the logs, instead of what is displayed in the devtools panel).

I think it will be a good time to enable these buttons after we implement [#27 :Save logs with/without filter applied](https://github.com/mozilla/extension-activity-monitor/issues/27). 

What do you think? Should we keep these buttons enabled here?

Reply:
@atiqueahmedziad I would be ok to defer it to a follow up if we have reasons to, but this activitylog page should still make it clear to the user that the view is filtered by tabId, and at that point how much is this different from what would happen for the activitylog page that we open in a tab if the user did set some filters on it?

Reply:
Yeah, the other filter options are visible and interact with the user (user has to click the checkbox to apply filter). But when we option the activitylog page in devtools, although user should know it filters the logs using current tab id, but the activitylog page's UI doesn't let the user know it has been filtered by tabId. 

I am adding a way to display in UI that activitylog page is filtered by tabId

Reply:
@rpl I made it is visible that, logs are being filtered by tab id. And enabled the menus. See the screenshot. Is this okay?

<img width="1280" alt="Screenshot 2020-08-14 at 8 08 46 PM" src="https://user-images.githubusercontent.com/8364578/90258297-2660d000-de6a-11ea-9e34-76659a9b3386.png">

Reply:
@rpl we are unable to save logs from devtools panel since `downloads` API is unavailable. I am getting the following error - 
`can't access property "onChanged", browser.downloads is undefined`

Reply:
Pass a message to the bg page and let the bg page call `browser.downloads.download`.